docs/docs/concepts/architecture.md

```diff
@@ -6,7 +6,7 @@ Dagger or Data Aggregator is a cloud native framework for processing real-time s
 
```

Comment:
Few general feedbacks:
* you will need to update readme as
* in the readme, and actually everywhere else, we are saying dagger is a real-time stream processing tool, we need to change that, maybe real-time and historical data processing?
* let's add a separate sub section under architecture section called Source, we can add details for Kafka and Parqeut and later BQ source can also come, as of now there is no starting point of our documentation for parquet source?

Reply:
> in the readme, and actually everywhere else, we are saying dagger is a real-time stream processing tool, we need to change that, maybe real-time and historical data processing?

I suppose this was done for the other sections in the earlier PR (https://github.com/odpf/dagger/pull/153). I will update the README and other places where not done.

> let's add a separate sub section under architecture section called Source, we can add details for Kafka and Parqeut and later BQ source can also come, as of now there is no starting point of our documentation for parquet source?

Makes sense. Let's add a page called `Configure a Source` where we can mention all the details about the sources. Then we can link this page in the `Create a Job` page under [this section](https://odpf.github.io/dagger/docs/guides/create_dagger#a-source)

Reply:
Fixed via commit https://github.com/odpf/dagger/pull/190/commits/eb9cc0bd3f455e661ab4687ad255ebbeacaff6ba and https://github.com/odpf/dagger/pull/190/commits/4fe1e05f3d368377b3f90d9d27efc1e1af69224e